Examples/Examples-macOS/Pods/Manifest.lock

```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: bbe5708b3a7f2dd7c70bb14225a7e37f67e1de58
 
-COCOAPODS: 1.4.0.beta.1
+COCOAPODS: 1.3.1
```

Comment:
please revert these changes. (use CP 1.4.0 beta)

Reply:
Not that it would have made a difference for this change but should that be reflected in the Gemfile?